src/linker/Linker.Steps/ProcessLinkerXmlBase.cs

```diff
@@ -168,6 +168,10 @@ protected virtual void ProcessTypes (AssemblyDefinition assembly, XPathNavigator
 
 				TypeDefinition type = assembly.MainModule.GetType (fullname);
 
+				// Before giving up, check .NET style nested type naming
```

Comment:
I'm not convinced this is a useful change. Why support it only for one of the possible XML inputs ?

Reply:
`<type fullname="Namespace.Type/Nested*" />` handled in `ProcessTypePattern` above or `<method signature="SomeType/NestedType SomeMethod()" />`.

Reply:
Good point.  Didn't think about those.

It seems like the same approach could be used in those places as well.

I'm not wed to this idea.  Someone internally mentioned it to me.  I think what happened was

1) They copied a nested type name from somewhere where it was in the .net style and pasted it into a link.xml file.
2) Linker issues a warning about no type found
3) Unity doesn't report any linker warnings.  So the warning goes unnoticed.
4) The person had to troubleshoot and figure out why their preservation had no effect.  I'm sure this was time consuming and confusing.

Given that (3) makes this scenario considerably more frustrating.  And (3) is on us for poor integration.  I can see how this scenario may be less problematic to the larger .net community.

If I implemented support in the other places would you want this change?  If not.  That's OK.